wails-frontend/src/pages/backupKeyPairs.tsx

```diff
@@ -0,0 +1,116 @@
+import { useState } from 'react';
+import { promptRequest } from '../events/events';
+import { useLocation } from 'react-router-dom';
+import { handleCancel } from '../utils/utils';
+import { Password, Button } from '@massalabs/react-ui-kit';
+import { FiCopy } from 'react-icons/fi';
+
+function BackupKeyPairs() {
+  const [inputPassword, setInputPassword] = useState<string | undefined>(
+    undefined,
+  );
+  const [show, setShow] = useState<boolean>(false);
+  const [privateKey, setPkey] = useState<string | undefined>(undefined);
+
+  let { state } = useLocation();
+  const req: promptRequest = state.req;
+
+  const baselineStr = 'Enter your password to show your private key';
+
+  function handleCopy() {
+    console.log('copy');
+  }
+
+  function handleShow() {
+    if (!inputPassword || !inputPassword.length) {
+      return;
+    }
+
+    if (!privateKey) {
+      // TODO: validate private key
+      setPkey(inputPassword);
+      setShow(!show);
+      setInputPassword('');
+      return;
+    }
+  }
```

Comment:
I am having hard time to understand the logic here! What we are trying to show/hide here!? 🤔 ... and why this apply only when is not privateKey!?

Reply:
![image](https://github.com/massalabs/thyra-plugin-wallet/assets/72019005/89e43cbf-9675-4b5f-9273-bf7f5fb19ebf)

@mazmassa This is the logic for this screen to show password. But i think we can replace the simple input by the password component.

Reply:
But your right! It's not clear for me too.

Reply:
> @mazmassa This is the logic for this screen to show password. But i think we can replace the simple input by the password component.

That's what I am asking myself... We have an issue from designs I guess! Let me double check with @LucasF243

Reply:
We have a new content then, closing here!